rebar.config

```diff
@@ -6,11 +6,37 @@
     {platform_define, "^[2-9]", 'POST_OTP_18'}
 ]}.
 
+{project_plugins, [covertool]}.
+
 {profiles, [
     {test, [
-        {deps, [{erlware_commons, "1.2.0"}, {elli, "3.0.0"}, {jsone, "1.4.7"}, {meck, "0.8.12"}]},
+        {overrides, [{override, rebar3,[{deps, [{erlware_commons, "1.2.0"}]}]}]},
+        {deps, [{hex_core, "0.4.0"}, {rebar, {git, "git://github.com/erlang/rebar3.git", {tag, "3.8.0"}}},
+                {erlware_commons, "1.2.0"}, {elli, "3.0.0"},
+                {jsone, "1.4.7"}, {meck, "0.8.12"}]},
         {erl_opts, [nowarn_export_all]}
     ]}
 ]}.
 
+{xref_ignores, [
```

Comment:
Just set the xref checks to:

```
{xref_checks,[undefined_function_calls,undefined_functions,locals_not_used,
              deprecated_function_calls,deprecated_functions]}.
```

instead of listing all these ignores.

Reply:
Yeah, I was trying to shoot for specificity (i.e., only ignoring what should be ignored), but I suppose that's most of everything.